### PR TITLE
Base: Add new Mouse + MouseSettings help

### PR DESCRIPTION
**Pull request by djwisdom**
### Mouse.md
image

### MouseSettings.md
image

### CODE REVIEW

File: Base/usr/share/man/man1/Mouse.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Synopsis
+
+```**sh
+$ Mouse
+```
+
+## Description
+
+`Mouse` is an application that displays interactive graphical mouse.
```
Comment (linusg) [inline Base/usr/share/man/man1/Mouse.md:15]:
```suggestion
Mouse is an application that displays interactive graphical mouse.
```

Backticks are reserved for code, this is not code :)

File: Base/usr/share/man/man1/Mouse.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+It simulates mouse clicks respective of the physical button pressed. It detects and displays corresponding left button, middle button, right button clicks, middle scroll wheel movements. It also shows side mouse button clicks if your mouse supports it.
```
Comment (linusg) [inline Base/usr/share/man/man1/Mouse.md:17]:
```suggestion
It simulates mouse clicks respective of the physical button pressed. It detects and displays corresponding left button, middle button, right button clicks, and middle scroll wheel movements. It also shows side mouse button clicks if supported by the connected mouse.
```

File: Base/usr/share/man/man1/MouseSettings.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+## Name
+
+![Icon](/res/icons/16x16/app-mouse.png) MouseSettings - MouseSettings application
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:3]: Application names should still be stylized with spaces :)

```suggestion
![Icon](/res/icons/16x16/app-mouse.png) Mouse Settings - Mouse settings application
```

File: Base/usr/share/man/man1/MouseSettings.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Synopsis
+
+```**sh
+$ MouseSettings
+```
+
+## Description
+
+`MouseSettings` is an application that displays advance mouse properties for user configuration.
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:15]:
```suggestion
Mouse Settings is an application that displays advanced mouse properties for user configuration.
```

File: Base/usr/share/man/man1/MouseSettings.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+
+Under Mouse tab, user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:18]:
```suggestion
Under the _Mouse_ tab, the user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).
```

File: Base/usr/share/man/man1/MouseSettings.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+
+Cursor Theme tab provides the user with drop-down list of available default cursor themes, Default and Dark.
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:21]:
```suggestion
The _Cursor Theme_ tab provides the user with a drop-down list of available default cursor themes.
```

Comment (linusg) [reply Base/usr/share/man/man1/MouseSettings.md:21]: Let's not assume which cursor themes are installed)

File: Base/usr/share/man/man1/MouseSettings.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+
+Supplemental cursors are available under Ports/serenity-theming app. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark among other things.
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:24]:
```suggestion
Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes include Durrque, Chillychilly, Jakande, Vanliga, and Vanliga-Dark, among others.
```

File: Base/usr/share/man/man1/MouseSettings.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+Cursor Highlight tab allows the user to change highlight color, opacity and size. The toggle key to enable and disable cursor highlight is Super+H.
+
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:28]:
```suggestion
The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
```

**Review by linusg — COMMENTED** [review]
Nice, couple of suggestions :)

File: Base/usr/share/man/man1/MouseSettings.md (hunk 6 above)
Comment (linusg) [reply Base/usr/share/man/man1/MouseSettings.md:21]: Actually it's probably fine, updated the suggestion.

File: Base/usr/share/man/man1/MouseSettings.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+The _Cursor Theme_ tab provides the user with drop-down list of available default cursor themes, Default and Dark.
+
+
+Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark, among others.
+
+
+The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
+
```
Comment (linusg) [inline Base/usr/share/man/man1/MouseSettings.md:28]: If you could remove the double newlines here that would be great :)

```suggestion
Mouse Settings is an application that displays advanced mouse properties for user configuration.

Under the _Mouse_ tab, the user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).

The _Cursor Theme_ tab provides the user with drop-down list of available default cursor themes, Default and Dark.

Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark, among others.

The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
```

**Review by linusg — COMMENTED** [review]
and now squash the fixup commits and it's good to go!

**Comment by djwisdom** [comment]
This is fun! :^)

**Pull request merged**